src/code42cli/output_formats.py

```diff
@@ -4,29 +4,29 @@
 
 import click
 
-from code42cli.cmds.enums import OutputFormat
 from code42cli.util import find_format_width
 from code42cli.util import format_to_table
 
 
-def output_format(_, __, value):
-    if value is not None:
-        if value == OutputFormat.CSV:
-            return to_csv
-        if value == OutputFormat.RAW:
-            return to_json
-        if value == OutputFormat.TABLE:
-            return to_table
-        if value == OutputFormat.JSON:
-            return to_formatted_json
-    # default option
-    return to_table
+CEF_DEFAULT_PRODUCT_NAME = "Advanced Exfiltration Detection"
+CEF_DEFAULT_SEVERITY_LEVEL = "5"
 
 
-def extraction_output_format(_, __, value):
+class OutputFormat:
+    TABLE = "TABLE"
+    CSV = "CSV"
+    JSON = "JSON"
+    RAW = "RAW-JSON"
+
+    def __iter__(self):
+        return iter([self.TABLE, self.CSV, self.JSON, self.RAW])
+
+
+def output_format(_, __, value):
     if value is not None:
+        value = value.upper()
         if value == OutputFormat.CSV:
-            return to_dynamic_csv
+            return to_csv
```

Comment:
This looks like a change, but the diff is just being funny. The regular `output_format()` uses `to_csv`, and the `extraciton_output_format` uses `to_dynamic_csv`